### DISCUSSION TITLE
Mining and storage independent or not

### DISCUSSION THREAD

**Discussion by Okan1919** [filecoin-project/community#345] [General] [answered]
Hi guys,
The mining process and storage process independent subject, or dependent?
There is no information on filecoin web pages and documents, or I didn' t understood ( mis something).
I couldn'T find any information. 
If them independent, I' ll some gain with my cheapest ISP' s Gbps network :D (lol its only 11$/m , I' m already pay internet for my daily useage. So connection free for me)
Thank for your reply

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
they are connected. no storage providing without "mining"
